python/paddle/v2/reader/creator.py

```diff
@@ -98,14 +102,17 @@ def recordio(paths, buf_size=100):
     if host_name not in os.environ.keys():
```

Comment:
For the line `host_name = "MASTER_SERVICE_HOST"` we don't have a "master service" currently.